src/python/vdo-setup-dmtest/source/Logger.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+  Logger - Sets up the vdo_setup_dmtest logger
+
+  Copyright (c) 2023 Red Hat
+"""
+
+from __future__ import print_function
+
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+logger = logging.getLogger('vdo-setup-dmtest')
+logger.setLevel(logging.DEBUG)
+
+from .Utils import runCommand
+
+class CustomFormatter(logging.Formatter):
+    yellow   = "\x1b[38;5;226m"
+    red      = "\x1b[38;5;196m"
+    bold_red = "\x1b[31;1m"
+    reset    = "\x1b[0m"
+
+    def __init__(self, fmt):
+        super().__init__()
+        self.fmt = fmt
+        self.FORMATS = {
+            logging.DEBUG: self.fmt,
+            logging.INFO: self.fmt,
+            logging.WARNING: self.yellow + self.fmt + self.reset,
+            logging.ERROR: self.red + self.fmt + self.reset,
+            logging.CRITICAL: self.bold_red + self.fmt + self.reset
+        }
+
+    def format(self, record):
+        log_fmt = self.FORMATS.get(record.levelno)
+        formatter = logging.Formatter(log_fmt)
+        return formatter.format(record)
```

Comment:
This appears to be using tab (or 4-space) indents.  The rest of the script(s) appear to be using 2-space indents.  Please adjust.

Reply:
Fixed in latest push.